content/departments/product-engineering/engineering/career-development/ic5/cloud-org.md

```diff
@@ -0,0 +1,42 @@
+# IC5 profile: Cloud Org
+
+See also [internal profile](https://docs.google.com/document/d/10e2s_X2P53Q9s17jdPpYoLdxwada74RbEGFtNSYT918/edit#)
+
+## Business need
+
+The Cloud business is of strategic interest to Sourcegraph. This role is responsible for setting technical direction for Cloud Org and coordination of cross-team / cross-org initiatives required to 1) launch Cloud as a paid Sourcegraph offering for SMB and 2) evolve it to become a mature SaaS product capable of satisfying the needs of all Sourcegraph customers that don’t want to manage On-Prem Sourcegraph deployments.
```

Comment:
Given the proposed changes in (private link) https://docs.google.com/document/d/1E1cRQONS1JyTy379RO9IOzZ4syr7tD-t8IUEVonvaSA/edit#heading=h.1g57y9fu65hc, would this formulation be more accurate?

```suggestion
The Cloud business is of strategic interest to Sourcegraph. This role is responsible for setting technical direction for Cloud Org and coordination of cross-team / cross-org initiatives required to 1) launch a paid Sourcegraph offering for Commercial customers and 2) evolve it to become a mature SaaS product capable of satisfying the needs of all Sourcegraph customers that don’t want to manage On-Prem Sourcegraph deployments.
```